### PR TITLE
Fix values in graphs

### PR DESCRIPTION
**Pull request by EyalMecilati**
not ready

### CODE REVIEW

File: cypress/fixtures/grid/titlesSymbolsObj.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        "Inferred":"Inferred (oz,k)",
+        "Resources":"Resources (oz,k)",
+        "Proven":"Proven (oz,k)",
+        "Probable":"Probable (oz,k)",
+        "P&P":"P&P (oz,k)",
+        "EV to FCF":"EV to FCF 1x",
+        "Price to NPV":"Price to NPV 1x",
+        "EV to Resources":"EV to Resources 1x",
+        "EV to M&I":"EV to M&I 1x",
+        "EV to Reservs":"EV to Reservs 1x",
```
Comment (alissaVrk) [inline cypress/fixtures/grid/titlesSymbolsObj.json:34]: there is no reason for the '1'..

File: cypress/integration/app/grid/gridColumnsVisibility.spec.js [hunk 2]
```diff
@@ -89,9 +90,11 @@ context("grid columns visibility", () => {
         return hiddenIndexes
       }
 
-    function validateHeaderAndContentCols(titles, row, hiddenFields = {}) {
+    function validateHeaderAndContentCols(titles, row,titlesSymbolsObj,hiddenFields = {}) {
       function testCell($cell, index, valuesCollection, titleOrRow) {
-        expect($cell.text().trim()).to.eq(valuesCollection[index]);
+        const valueText = valuesCollection[index]
+        const titleWithSymbol =  titlesSymbolsObj[valueText] ? titlesSymbolsObj[valueText] : valueText;
```
Comment (alissaVrk) [inline cypress/integration/app/grid/gridColumnsVisibility.spec.js:96]: change that name of the const, it's once a title and once a cell value..
very hard to understand

File: src/comps/grid/useGridColumns.tsx [hunk 3]
```diff
@@ -24,12 +25,16 @@ import { Link, useLocation } from "@reach/router";
 const cols = grid_texts.columns as json;
 
 function getColumnBaseProps(
-    fieldName: keyof CompanySummery
+    fieldName: keyof CompanySummery,
+    companiesData:{companies: CompanySummery[], productAndCurrency: ProductsPricesObj}
```
Comment (alissaVrk) [inline src/comps/grid/useGridColumns.tsx:29]: please pass only what you need - the product and currency

File: src/comps/grid/useGridColumns.tsx [hunk 4]
```diff
@@ -93,17 +98,17 @@ export function useGridColumns(colsCategs: Array<ColumnCategory>, companiesData:
...
           filteredValue: getFilterValue(fieldName, userPrefsFilters, companiesData.productAndCurrency),
           render: (value,record) => {
             if(fieldName === 'name'){
               return <Link id={`#goTo${value}Page`} to={'/company-page/'+record.id+isQa}>{value}</Link>
             }
-            return value && formatValueByType(fieldDef, value, {
-              productType: companiesData.productAndCurrency.productType, 
-              currencyType: companiesData.productAndCurrency.currencyType
-            });
+            if(fieldName === 'nextMilestoneQuarter' || fieldName === 'targetCommercialProduction'){
```
Comment (alissaVrk) [inline src/comps/grid/useGridColumns.tsx:108]: please do this by type and not by name

**Review by alissaVrk — CHANGES_REQUESTED** [review]
not done yet

File: cypress/fixtures/grid/titlesSymbolsObj.json (hunk 1 above)
Comment (EyalMecilati) [reply cypress/fixtures/grid/titlesSymbolsObj.json:34]: tal told me that's how he wanted it, (1 week ago)

File: cypress/fixtures/grid/singleRow.json [hunk 5]
```diff
@@ -444,43 +444,43 @@
...
+    "800",
+    "3.68",
+    "4.74",
+    "8.42",
+    "0.05",
+    "8.47",
+    "0.05",
+    "0.05",
+    "0.09",
+    "0.25 1x",
```
Comment (alissaVrk) [inline cypress/fixtures/grid/singleRow.json:475]: remove the '1', no '()' no space between the number and the 'x'

File: cypress/integration/app/grid/gridColumnsVisibility.spec.js [hunk 6]
```diff
@@ -89,9 +90,11 @@ context("grid columns visibility", () => {
         return hiddenIndexes
       }
 
-    function validateHeaderAndContentCols(titles, row, hiddenFields = {}) {
+    function validateHeaderAndContentCols(titles, row,titlesSymbolsObj,hiddenFields = {}) {
       function testCell($cell, index, valuesCollection, titleOrRow) {
-        expect($cell.text().trim()).to.eq(valuesCollection[index]);
+        const cellOrTitle = valuesCollection[index]
+        const titleWithSymbol =  titlesSymbolsObj[cellOrTitle] ? titlesSymbolsObj[cellOrTitle] : cellOrTitle;
```
Comment (alissaVrk) [inline cypress/integration/app/grid/gridColumnsVisibility.spec.js:96]: naming.. it is really hard to understand
the general case is cellContent and if applicable translate the value (symbol)

File: src/comps/companyPage/multiplierPage.tsx [hunk 7]
```diff
@@ -9,20 +9,20 @@ import {
...
     <Card title="Company Multiplier Info">
       <Descriptions size="small" column={3}>
-          {printDescriptionsItemByValue(summery.evToFcf,'evToFcf')}
-          {printDescriptionsItemByValue(summery.priceToNpv,'priceToNpv')}
-          {printDescriptionsItemByValue(summery.evToResources,'evToResources')}
-          {printDescriptionsItemByValue(summery.evToM_and_I,'evToM_and_I')}
-          {printDescriptionsItemByValue(summery.evToReserves,'evToReserves')}
-          {printDescriptionsItemByValue(summery.evToLom,'evToLom')}
-          {printDescriptionsItemByValue(summery.npvToInitialCapex,'npvToInitialCapex')}
+          {printDescriptionsItemByValue(summery.evToFcf,'evToFcf',' 1x')}
```
Comment (alissaVrk) [inline src/comps/companyPage/multiplierPage.tsx:19]: same

File: src/comps/grid/useGridColumns.tsx [hunk 8]
```diff
@@ -93,17 +99,20 @@ export function useGridColumns(colsCategs: Array<ColumnCategory>, companiesData:
...
             }
-            return value && formatValueByType(fieldDef, value, {
-              productType: companiesData.productAndCurrency.productType, 
-              currencyType: companiesData.productAndCurrency.currencyType
-            });
+            if(fieldDef.dataType === FieldDataTypes.Quorter){
+              return getFormatQuorter(value)
+            }
+            if(fieldDef.numberType === NumberType.Multi){
+              return value && value + ' 1x'
```
Comment (alissaVrk) [inline src/comps/grid/useGridColumns.tsx:113]: same

**Review by alissaVrk — CHANGES_REQUESTED** [review]

**Review by alissaVrk — APPROVED** [review]

**Pull request merged**